### DISCUSSION TITLE
Does this expose an API adequate for MALSync integration?

### DISCUSSION THREAD

**Discussion by beedell-roke** [Kurozora/kurozora-web#390] [Q&A] [answered]
Does Kurozora expose an API adequate for integrating with https://github.com/MALSync/MALSync/blob/5f11f5ff22e81b602901af79fb04cbef427926ae/README.md#mal-sync, as https://github.com/MALSync/MALSync/issues/2359#issue-2217178469 requests? ~~I ask at here rather than https://github.com/Kurozora/KurozoraKit/discussions/ because it isn't enabled.~~

**Comment by kiritokatklian** [comment, marked as answer]
Yes! I've also talked to the devs on 2023-12-26 about integrating Kurozora, but nothing came out of it. Unfortunately I don't have the time to dive into their project and add the integration myself, so it's up to them and their users to decide what to do next.
